packages/core/src/products/details/client/getColorGrouping.js

```diff
@@ -19,6 +19,10 @@ import join from 'proper-url-join';
  * @function getColorGrouping
  * @memberof module:products/client
  *
+ * @deprecated Since version 1.x.x.
+ * Will be deleted in version 2.0.0.
+ * In case you need this behavior, use the .
```

Comment:
Will this text be updated before release?

Reply:
fixed